### ISSUE TITLE
'wpt install firefox driver' fails for OSX

### ISSUE THREAD

**Issue by lukebjerring** [w3c/web-platform-tests#10143]
`requests.exceptions.SSLError: HTTPSConnectionPool(host='github.com', port=443): Max retries exceeded with url: /mozilla/geckodriver/releases/download/v0.20.0/geckodriver-v0.20.0-macos.tar.gz (Caused by SSLError(SSLError(1, u'[SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:661)'),))`

Likely due to OpenSSL version.

FWIW, 
    python -c "import ssl; print(ssl.OPENSSL_VERSION)"
    OpenSSL 0.9.8zh 14 Jan 2016

**Comment by jgraham** [comment]
Right, I don't know what we could do here other than print a more specific error. The problem is that OSX ships with an outdated version of OpenSSL that is no longer supported by GitHub. You need to install Python via some other means to use it.

**Comment by lukebjerring** [comment]
I mostly opened this issue for discoverability/search reasons if others encounter the same thing.
This issue/error is essentially a dead end. There's a few possible options:

- More specific error
- instructions on where to get geckodriver manually
  - Also, we should (and might already?) bypass the install when geckodriver is already on the path
- instructions on installing python2.7 via something else e.g. brew
